### PR TITLE
Django Tutorial Part 8: User authentication and permissions

### PR DESCRIPTION
**Pull request by trungthanhnguyen64**
ready

### CODE REVIEW

**Comment by trungthanhnguyen64** [comment]
ready

**Comment by trungthanhnguyen64** [comment]
ready

File: catalog/models.py [hunk 1]
```diff
@@ -70,20 +53,27 @@ class BookInstance(models.Model):
...
+        permissions = (("can_mark_returned", "Set book as returned"),)
 
     def __str__(self):
         """String for representing the Model object."""
         return f'{self.id} ({self.book.title})'
+    @property
+    def is_overdue(self):
+        if self.due_back and date.today() > self.due_back:
+            return True
+        return False
```
Comment (huongnt-2545) [inline catalog/models.py]:
```suggestion
        return self.due_back and date.today() > self.due_back
```

**Comment by trungthanhnguyen64** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
mợt pull nhé

**Comment by huongnt-2545** [comment]
merged

**Pull request merged**